<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>fm_eventos</class>
 <widget class="QDialog" name="fm_eventos">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>324</width>
    <height>530</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gestão de eventos</string>
  </property>
  <widget class="QPushButton" name="pushButton_fechar">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>496</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Fechar</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget_eventos">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>301</width>
     <height>341</height>
    </rect>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>301</width>
     <height>131</height>
    </rect>
   </property>
   <property name="title">
    <string>Inclusão / Alteração / Exclusao</string>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>27</y>
      <width>71</width>
      <height>27</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>ID:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_id">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>89</x>
      <y>27</y>
      <width>201</width>
      <height>27</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Título:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_titulo"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>60</y>
      <width>281</width>
      <height>27</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Descrição:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_descricao"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>94</y>
      <width>281</width>
      <height>27</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QPushButton" name="pushButton_salvar">
       <property name="text">
        <string>Salvar</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_excluir">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Excluir</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_cancelar">
       <property name="text">
        <string>Cancelar</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QLabel" name="label_numero_da_linha">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>500</y>
     <width>52</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>lineEdit_id</tabstop>
  <tabstop>lineEdit_titulo</tabstop>
  <tabstop>lineEdit_descricao</tabstop>
  <tabstop>pushButton_salvar</tabstop>
  <tabstop>pushButton_excluir</tabstop>
  <tabstop>pushButton_cancelar</tabstop>
  <tabstop>tableWidget_eventos</tabstop>
  <tabstop>pushButton_fechar</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
